sycl/include/CL/sycl/detail/scheduler/commands.hpp

```diff
@@ -159,19 +170,38 @@ class Command {
   unsigned MLeafCounter = 0;
 
   const char *MBlockReason = "Unknown";
+
+  // Removing guard for the member variables as this
+  // gives freedom for all tests to not have to know whether
+  // the runtime has been built with instrumentation or
+  // not :#ifdef XPTI_ENABLE_INSTRUMENTATION
```

Comment:
Suggest rewriting to something like:
Do not guard the member variables below with XPTI_ENABLE_INSTRUMENTATION
to have the same object layout when the macro in the library and a SYCL app are
not the same.

Reply:
Done!